bin/check-all.sh

```diff
@@ -21,7 +21,7 @@
 
 set -e
 
-SCALAs=( "2.10" )
+SCALAs=( "2.10", "2.11" )
 JDKs=( "JAVA7_HOME" "JAVA8_HOME" )
```

Comment:
I realized that JAVA7_HOME should be removed here. Creating another ticket for this.